docs/content/latest/explore/observability/grafana-dashboard/macos.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Grafana Dashboard
+headerTitle: Grafana Dashboard
+linkTitle: Grafana Dashboard 
+description: Learn about setting up the community dashboards with Prometheus data source using Grafana.
+menu:
+  latest:
+    identifier: observability-1-macos
+    parent: explore-observability
+    weight: 240
+isTocNested: true
+showAsideToc: true
+---
+
+ <ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="/latest/explore/observability/grafana-dashboard/macos/" class="nav-link active">
+      <i class="fab fa-apple" aria-hidden="true"></i>
+      macOS
+    </a>
+  </li>
+  </ul>
+   
+  You can create dashboards for your local YugabyteDB clusters using [Grafana](https://grafana.com/grafana/), an open source platform to perform visualization and analytics which lets you add queries and explore metrics to customize your graphs.
+  To create dashboards, you need to add Prometheus metrics as a data source in Grafana. The visualization it provides gives a better understanding of the health and performance of your YugabyteDB clusters. 
+
+  This tutorial uses the [yugabyted](https://docs.yugabyte.com/latest/reference/configuration/yugabyted) local cluster management utility. 
+  The purpose of this tutorial is to utilize the Prometheus metrics obtained from following [Prometheus Integration](https://docs.yugabyte.com/latest/explore/observability/prometheus-integration/macos/) and then  creating dashboards via Grafana.
+  ## Prerequisite
+
+  A local instance of Prometheus displaying the metrics of your YugabyteDB cluster is required. To do this,
+  - Create a local instance of Prometheus and perform  the Steps 1-4 listed under [Prometheus Integration](https://docs.yugabyte.com/latest/explore/observability/prometheus-integration/macos/). (Step 5 is optional)
+
+  ## 1. Setup Grafana and add Prometheus as a data source.
+  - Install Grafana and start the server from the [Grafana documentation](https://grafana.com/docs/grafana/latest/installation/mac/).
+  - Open the Grafana UI on http://localhost:3000. The default login is `admin` / `admin`. 
+  - Follow the steps to create a Prometheus data source from this [page](https://prometheus.io/docs/visualization/grafana/). 
+
+  ## 2. Create a dashboard
+  There are different ways to create a dashboard in Grafana. For this tutorial , you will need to use the import option and pick the Yugabytedb community dashboard present under the list of Grafana [Dashboards](https://grafana.com/grafana/dashboards/12620).
+
+  - On the Grafana UI, select the `+` icon and choose the import option.
+
+  ![Grafana import](/images/ce/grafana-add.png)
```

Comment:
In the build preview (https://deploy-preview-8924--infallible-bardeen-164bc9.netlify.app/latest/explore/observability/grafana-dashboard/macos/) these aren't indented at the same level as the bulleted text that they belong to. Once you've fixed the indentation of all the text on the page, make sure the images are indented relative to their bullet paragraphs so they align properly.

Reply:
Hopefully got this one right now.